app/customer.py

```diff
@@ -0,0 +1,48 @@
+from math import sqrt
+
+
+class Customer:
+    def __init__(
+            self,
+            name: str,
+            product_cart: dict,
+            location: list,
+            money: int,
+            car: dict) -> None:

```

Comment:
The `car` parameter should be an instance of the `Car` class, not a dictionary. This will allow you to access the `fuel_consumption` attribute directly from the `Car` instance.